code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -117,20 +120,20 @@
 
 #undef VAMP_DRAIN_AMOUNT
 
+
+/mob/living/carbon/Stat()
+	..()
+	if(statpanel("Status"))	
+		var/obj/item/organ/heart/vampire/darkheart = getorgan(/obj/item/organ/heart/vampire)
+		if(darkheart)
```

Comment:
Check for vampire status with `isvampire()` instead of getting their heart type.

Reply:
having a vampire heart should allow you to see your blood level, no?

Reply:
mechanic starts even on heart transplants not just with vampire species type.

Reply:
this is intended.

Reply:
I wasn't aware, so it's fine.